Unshaky/ShakyPressPreventer.m

```diff
@@ -57,58 +57,73 @@ - (void)loadIgnoreExternalKeyboard {
     ignoreExternalKeyboard = [defaults boolForKey:@"ignoreExternalKeyboard"]; // default No
 }
 
-- (CGEventRef)filterShakyPressEvent:(CGEventRef)event {
```

Comment:
Also, I don't see why this line has been changed...

Reply:
Has it even been changed?  I don't understand why it's been flagged.